src/radical/pilot/task_overlay/worker.py

```diff
@@ -217,6 +217,29 @@ def _call(self, data):
         return out, err, ret
 
 
+    # --------------------------------------------------------------------------
+    #
+    def _mpi(self, data):
```

Comment:
there is an inconsistency in this method (with loaded modules and used corresponding methods), and I wasn't sure what's inside variable `data` and it should be input for `rpa.LaunchMethod.create(...`?

p.s. may be move this method to another PR?

Reply:
Hmm, how did that get in here indeed - that's unrelated I think...  Let me check what happened, will move it out.

Reply:
An update from devel solved this :-)